src/components/Dashboard/DonationHistories/DonationHistories.tsx

```diff
@@ -116,6 +116,12 @@ const DonationHistories = ({
   const goalColor = palette.graphite.main;
   const averageColor = palette.graphite.main;
   const pledgedColor = palette.yellow.main;
+  const goalLineStyles = {
+    stroke: goalColor,
+    strokeDasharray: '5,8',
+    strokeLinecap: 'round' as const,
```

Comment:
Do we need the type casting?

Reply:
The type for `strokeLinecap` is `"butt" | "round" | "square" | "inherit"`, so we have to use `as const` so that TypeScript interprets `'round'` as a literal string and not just `string`.